torchao/prototype/moe_training/scaled_grouped_mm.py

```diff
@@ -48,7 +48,7 @@ def _scaled_grouped_mm(
     """
     # TODO: Remove logging once prototype is more mature. This is currently very useful for development and debugging.
     if scaling_type == MoEScalingType.FP8_ROWWISE:
-        # print("Using fp8 rowwise scaled_grouped_mm")
+        print("Using fp8 rowwise scaled_grouped_mm")
```

Comment:
!

Reply:
we shouldnt be printing though.. maybe logging but forsure not printing

Reply:
@drisspg I landed https://github.com/pytorch/ao/pull/2835 and rebased on top of main, so the print statements are logging now.